extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -138,13 +146,19 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
     });
   }
 
-  /**
-   * @private
-   */
+  /** @private */
   onClick_() {
     this.storeService_.dispatch(Action.ADD_SHOPPING_DATA, {
       'activeProductData': this.tagData_,
     });
+    this.openPageAttachment_();
+  }
+
+  /** @private */
+  openPageAttachment_() {
+    whenUpgradedToCustomElement(this.pageAttachmentEl_)
+      .then(() => this.pageAttachmentEl_.getImpl())
+      .then((impl) => impl.open());
```

Comment:
Can you link me to where the attachment knows what content to render when opening? I don't like that it's happening in two different places, we should think of how to make this better